_events/2019-03-25-first-galantries-workshop.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: event
+title: 'RNA-seq workshop for beginners: from sequences to visualization using Galaxy and R'
+starts: 2019-05-06
+ends: 2019-05-08
+locations:
+- 
+    name: Erasmuc MC
+    street: Wytemaweg 80
+    postal: 3015 CN
+    city: Rotterdam
+    country: The Netherlands
+- 
+    name: Rechenzentrum
+    street: Hermann-Herder-Straße 10
+    postal: 79104
+    city: Freiburg
+    country: Germany
+    geo:
+        lat: 48.003799
+        lon: 7.848305
+---
+
+The [Gallantries team]({% link team.md %}) is offering its first RNA-seq workshop for beginners on May, 6-8th.
```

Comment:
```suggestion
The [Gallantries team]({% link team.md %}) is offering its first RNA-seq workshop on May, 6-8th.
```

Reply:
Probably not super important that it's for beginners?

Reply:
I think it is relevant. It specifies the audience of the workshop.